src/status_im/common/privacy_mode/events.cljs

```diff
@@ -0,0 +1,12 @@
+(ns status-im.common.privacy-mode.events
+  (:require [status-im.common.privacy-mode.view :as privacy-mode]
+            [utils.re-frame :as rf]))
+
+(rf/reg-event-fx :privacy-mode/toggle-privacy-mode
```

Comment:
When support for this feature is added in status-go, we will very likely need to logout the user after toggling the setting. Just something to keep an eye on.